java/src/main/resources/template.properties

```diff
@@ -184,3 +184,25 @@ pubsubtogcs.gcs.output.data.format=
 dataplex.gcs.bq.target.dataset=<bq target dataset>
 dataplex.gcs.bq.save.mode="errorifexists"
 dataplex.gcs.bq.incremental.partition.copy="yes"
+
+# Kafka To BQ
+## Kafka servers
+kafka.bq.bootstrap.servers=<kafka broker list>
+## Kafka topics
+kafka.bq.topic=<kafka topic names>
+## BigQuery output dataset
+kafka.bq.dataset=<output bigquery dataset>
+## BigQuery output table
+kafka.bq.table=<output bigquery table>
+## GCS bucket name, for storing temporary files
+kafka.bq.temp.gcs.bucket=<gcs bucket name>
+## GCS location for maintaining checkpoint
+kafka.bq.checkpoint.location=<gcs bucket location maintains checkpoint>
+## Offset to start reading from
+kafka.bq.starting.offset=earliest
+## Waits for specified time in ms before termination of stream
+kafka.bq.await.termination.timeout=420000
+## Fails the job when data is lost. Accepted values: true, false
+kafka.bq.fail.on.dataloss=false
+## Ouptut mode for writing data. Accepted values: 'append', 'complete', 'update'
+kafka.bq.steam.output.mode=append
```

Comment:
@shashank-google This property justifies **what to write** rather than **how to write** to the sink.
It governs the process of writes, for the new data as and when it arrives in the dataset. 

Modified the README.md to be more descriptive on these new spark properties. Please review.